eng/Npm.Workspace.nodeproj

```diff
@@ -4,6 +4,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.props" />
 
   <PropertyGroup>
+    <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
```

Comment:
Was this being defaulted to `true` for this project?

Reply:
Yes, I think we need to fix this "long term" so that we don't have to do anything special inside SourceBuild.props